lxd/api_project.go

```diff
@@ -276,6 +282,8 @@ func projectPatch(d *Daemon, r *http.Request) response.Response {
 		project.Description,
 		project.Config["features.images"],
 		project.Config["features.profiles"],
+		project.Config["limits.containers"],
+		project.Config["limits.virtual-machines"],
```

Comment:
Etag should work just fine if you feed it `project.Config` as a whole rather than having to pass the individual keys.
I don't know why this was done this way in the first place but it's not how we're doing it elsewhere.